samples/samples-js/AddProductWithMultiplePrimaryColumnsAndIdentity/function.json

```diff
@@ -0,0 +1,28 @@
+{
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "name": "req",
+      "direction": "in",
+      "type": "httpTrigger",
+      "methods": [
+        "get",
+        "post"
```

Comment:
Just GET, not POST

(please go through all the functions you've made and verify that they match the method types of the corresponding C# functions. We should follow up later on whether we should be using GET or POST for each one, but for now let's at least keep them consistent)